packages/design-system-react/package.json

```diff
@@ -48,14 +48,25 @@
   },
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
+    "@metamask/design-system-tailwind-preset": "workspace:^",
```

Comment:
Once `@metamask/design-system-tailwind-preset` has been published we will move it from `devDependencies` to `peerDependencies` issue here: https://github.com/MetaMask/metamask-design-system/issues/74